bin/install-then-build.sh

```diff
@@ -24,13 +25,15 @@ fi
 
 # Build x86_64 Mercury.
 rm -rf "$SRC_DIR"
-tar xvf "$DLD_DIR/rotd-$MERCURY_ROTD.tar.gz" -C "$DLD_DIR"
+tar xf "$DLD_DIR/rotd-$MERCURY_ROTD.tar.gz" -C "$DLD_DIR"
 
 X86_64_TRIPLE=x86_64-apple-darwin
 
 pushd "$SRC_DIR"
 
-./configure \
+# Explicitly use zsh to workaround a parse error that occurs with bash 3.2.
+# (This can be reverted in the next Mercury upgrade.)
+zsh ./configure \
```

Comment:
zsh became the [default shell in macOS Catalina](https://support.apple.com/en-us/102360) so it might be worth just leaving it in the future (it's unlikely they will ever update to a GPL3 version of bash).